assets/notification-utils.js

```
@@ -0,0 +1,82 @@
+// Shared notification utility functions
+const NotificationUtils = {
+    // Get notification icon class with customizable size
+    getNotificationIconClass(notification, size = 'w-8 h-8') {
+        const baseClass = `${size} rounded-full flex items-center justify-center`;
+        
+        switch (notification.type) {
+            case 'error':
+                return baseClass + ' bg-error/20 text-error';
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)